content/docs/command-reference/import-url.md

```diff
@@ -179,6 +179,12 @@ produces a regular stage in `dvc.yaml`.
 
 - `-h`, `--help` - prints the usage/help message, and exit.
 
+- `--version-aware` - capture cloud versioning information when importing the
+  file. By default, DVC will automatically capture cloud versioning information
+  if the URL contains a cloud versioning ID. When `--version-aware` is provided
+  along with a URL that does not contain a cloud versioning ID, DVC will capture
+  the latest version of the file.
```

Comment:
Let's also explain that dvc will pull that version from the source location even if it's overwritten, and will not push another copy of it to the remote.

cc @jorgeorpinel Is there somewhere in the data management user guide we want to this info also?

Reply:
We'll def. need UG updates to go over cloud versioning -- can't explain everything in an option text. For now I'd focus on what the flag does, and make some mentions/explanations in the cmd ref Description (which in this case is already super long and should be rewritten/ moved to UG eventually).

Reply:
p.s. Specifically, there's is some draft content about this in https://github.com/iterative/dvc.org/pull/4119/files#diff-d01612907e4ab14238625d537eaf42852d8566901d1bfe2fd3f2d4406a2d1dfc ATM.